src/ophys_etl/modules/neuropil_correction/schemas.py

```diff
@@ -0,0 +1,52 @@
+from argschema import ArgSchema, fields
+
+from ophys_etl.schemas.fields import H5InputFile
+
+
+class NeuropilCorrectionJobOutputSchema(ArgSchema):
+    neuropil_trace_file = H5InputFile(
+        required=True,
+        description=("Path to input h5 file containing neuropil traces"),
+    )
+
+    roi_trace_file = H5InputFile(
+        required=True,
+        description=("Path to input h5 file containing roi traces"),
+    )
+
+    storage_directory = fields.OutputDir(
```

Comment:
It's inconsistent to ask for `storage_directory` as well as individual output file `neuropil_correction`. I would suggest to either only use `storage_directory` or to specify the other output dir `neuropil_subtraction_plots_output_dir` in place of `storage_directory`

Reply:
See above, I want to clarify what might break if the arg schema is changed.

Reply:
So the output.json wouldn't even be needed if the output file is removed from the schema?

Reply:
the output schema not needed but it makes it clear what are the expected outputs

Reply:
I decided to add the individual output file and the plots dir to the output schema as I believe that makes it clearer what the outputs are.